pkg/netconfig/netconfig.go

```diff
@@ -76,11 +76,14 @@ func InitLoggingChain(ipVersion string) error {
 
 // firewaller
 
-func IPTablesLoggingChainRule(ipVersion string, protocol string, ipSet string, device string, check bool, blockIngress bool) error {
+func IPTablesLoggingChainRule(ipVersion string, protocol string, ipSet string, device string, check bool, delete bool, blockIngress bool) error {
```

Comment:
Not really great to have to boolean flags to determine the action. If both are `true` the result is not obvious for the caller what will happen.